composer.json

```diff
@@ -20,12 +20,12 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
-        "friendsofsymfony/oauth2-php": "~1.1",
-        "symfony/dependency-injection": "^5.3",
-        "symfony/framework-bundle": "^5.3",
-        "symfony/security-bundle": "^5.3",
-        "symfony/twig-bundle": "^5.3"
+        "php": "^8.1",
+        "klapaudius/oauth2-php": "^1.5",
```

Comment:
Can we just incorporate the necessary changes from this fork?  It doesn't seem like a whole lot, and some things can be left out.  [The comparison.](https://github.com/klapaudius/oauth2-php/compare/546f869d68fb79b284752e6787263d797165dba4..master)